### ISSUE TITLE
Using shift+k as scroll key

### ISSUE THREAD

**Issue by Toeplitz** [aioutecism/amVim-for-VSCode#210]
I'm attempting to use shift+k as a key to scroll up with the following code:

    {
        "key": "shift+k",
        "command": "extension.smoothscroll_cursorUp",
        "when": "amVim.mode == 'NORMAL'"
    }

What could be preventing this?

**Comment by Toeplitz** [comment]
@aioutecism: Yes, doing this it works as expected and executes extension.smoothscroll_cursorUp.

**Comment by aioutecism** [comment]
@Toeplitz 
Thank you for trying that.

Cloud you try this setting with amVim **enabled**?

```json
{
	"key": "shift+k",
	"command": "extension.smoothscroll_cursorUp",
	"when": "textInputFocus && amVim.mode != 'INSERT'"
}
```

**Comment by Toeplitz** [comment]
@aioutecism: Sorry, I misread your first comment. Let me try again.

With amVim disabled using "when": "textInputFocus", I just type capital K letters.

**Comment by Toeplitz** [comment]
Thanks it looks like this works. Part of what had me confused here is that I need to exit the keymaps file and save before the settings are updated.

**Comment by aioutecism** [comment]
@Toeplitz 
Oh, great!
And yes, settings only take effects after they are saved.
